First Python Program.py

```diff
@@ -0,0 +1,20 @@
+def MyAddFunction(my_first_argument, my_second_argument):
+  if (my_first_argument == my_second_argument):
+    #first code block
+    return ' Why???'
+  return my_first_argument + my_second_argument
+
+var1 = 'Hi1'
+var2 = 'Hi2'
+
+if False:
+    print('No')
```

Comment:
You should be using a buffer to write out and erase content on the screen. That would be a good milestone to clear and rewrite the screen each time you make a selection